util/testutil/integration/pins.go

```diff
@@ -0,0 +1,15 @@
+package integration
+
+var pins = map[string]map[string]string{
+	// busybox is pinned to 1.35. Newer produces has "illegal instruction" panic on some of Github infra on sha256sum
```

Comment:
Is this issue tracked somewhere?

Can't find a ticket in https://github.com/docker-library/busybox/issues

Reply:
I haven't reported to busybox yet. If you do, you can post this debug: https://github.com/tonistiigi/gh-busybox-sigill-debug/actions/runs/3858482123

Good CPUs: 8370C, E5-2673
Bad CPUs: 8171M, 8272CL

Reply:
https://bugs.busybox.net/show_bug.cgi?id=15236